DOCKER/README.md

```diff
@@ -31,8 +31,8 @@ To get started developing applications, see the [application developers guide](h
 A quick example of a built-in app and Tendermint core in one container.
 
 ```
-docker run -it --rm -v "/tmp:/tendermint" tendermint/tendermint init
-docker run -it --rm -v "/tmp:/tendermint" tendermint/tendermint node --proxy_app=kvstore
+docker run -it --rm -v "/tmp:/tendermint" dashpay/tenderdash init
+docker run -it --rm -v "/tmp:/tendermint" dashpay/tenderdash node --proxy_app=kvstore
```

Comment:
Not sure if we need this? 
```suggestion
docker run -it --rm -v "/tmp:/tenderdash" dashpay/tenderdash init
docker run -it --rm -v "/tmp:/tenderdash" dashpay/tenderdash node --proxy_app=kvstore
```

Reply:
Yeah, probably we don't want to rename homedirs, users, binary, and so on. I feel like, it's too many changes and so easy to break things. We could do the total renaming later when we have time to deal with it, right? :)))